propka/bonds.py

```diff
@@ -392,7 +391,8 @@ def has_bond(self, atom1, atom2):
             return True
         return False
 
-    def make_bond(self, atom1, atom2):
+    @classmethod
```

Comment:
ditto... `staticmethod`

Reply:
Addressed in https://github.com/Electrostatics/propka-3.1/commit/db8571414e01b7451790c54450f90aa875cbd180